### PR TITLE
Add readable terms for the operators

### PR DESCRIPTION
**Pull request by beerose**
### Description
In Browse rows, there were listed postgres operators. This PR adds readable english terms and the operator in brackets so that anybody can understand it quickly.

Screenshot 2020-01-14 at 11 38 03

### Affected components

- [x] Console

#### Breaking changes

- [x] No Breaking changes

### CODE REVIEW

**Comment by coco98** [comment]
@beerose @rikinsk Should we change `ilike` to `like (ignore case)` or `like (case insensitive)`?

**Comment by beerose** [comment]
Yeah, it makes sense since we want to make it much more readable. I think I'd rather go with `(case insensitive)` since it's more common. 

But take into account that it would quite widen the dropdown:

Screenshot 2020-01-14 at 12 19 43

**Comment by beerose** [comment]
I've changed dropdown options to be of the format `[<graphql equivalent operator>] logical name` and added `(case insensitive)` to `ilike` and `nilike`.

I also added the default value `%%` when `like`, `nlike`, `ilike` or `nilike` is chosen. Small demo:
likes

File: console/src/components/Services/Data/constants.js [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+  { name: 'not in', value: '$nin', graphqlOp: '_nin' },
+  { name: '>', value: '$gt', graphqlOp: '_gt' },
+  { name: '<', value: '$lt', graphqlOp: '_lt' },
+  { name: '>=', value: '$gte', graphqlOp: '_gte' },
+  { name: '<=', value: '$lte', graphqlOp: '_lte' },
+  { name: 'like', value: '$like', graphqlOp: '_like' },
+  { name: 'not like', value: '$nlike', graphqlOp: '_nlike' },
+  { name: 'like (case-insensitive)', value: '$ilike', graphqlOp: '_ilike' },
+  {
+    name: 'not ilike (case-insensitive)',
```
Comment (rikinsk) [inline console/src/components/Services/Data/constants.js]: `not ilike` -> `not like`

File: console/src/components/Services/Data/constants.js [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+    value: '$nilike',
+    graphqlOp: '_nilike',
+  },
+  { name: 'similar', value: '$similar', graphqlOp: '_similar' },
+  { name: 'not similar', value: '$nsimilar', graphqlOp: '_similar' },
```
Comment (rikinsk) [inline console/src/components/Services/Data/constants.js]: `_similar` -> `_nsimilar`

**Comment by beerose** [comment]
I handled the `$in` and `$nin` case. Also, in the future, we could think about adding validation -- validate if, for expected integer, there's a number, for expected array there's a valid array, etc.

**Pull request merged**